.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,34 +1,33 @@
 ---
-name: "General Issue"
-about: "Describe your issue here"
-title: "[Issue]: "
-labels: ["needs-triage"]
+name: "Report a Bug"
+about: "Report a bug in the OpenOps platform or its documentation"
```

Comment:
Not sure if documentation should be here at all, but if we keep it here we could change this to:

```suggestion
about: "Report a bug in the OpenOps platform or a problem with the documentation"
```

Reply:
Yea you're right I'll remove it